docs/examples/driver_examples/QCoDeS example with Galil DMC4133 Controller.ipynb

```diff
@@ -0,0 +1,895 @@
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:18:56.092696Z",
+     "start_time": "2021-09-06T12:18:56.052425Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connected to: Galil Motion Control, Inc. DMC4133 (serial:16639, firmware:1.3f) in 0.03s\n"
+     ]
+    }
+   ],
+   "source": [
+    "controller = DMC4133Controller(name='controller', address='192.168.8.74')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "89610c52",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:18:57.715235Z",
+     "start_time": "2021-09-06T12:18:57.705231Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "arm = Arm(controller)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c968a25",
+   "metadata": {},
+   "source": [
+    "Now that we have imported an initialized the controller and the arm. We need to calibrate the arm."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78869855",
+   "metadata": {},
+   "source": [
+    "# Calibration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d66d4b5",
+   "metadata": {},
+   "source": [
+    "Check the state of the motors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "id": "2442e7a1",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:46:08.312660Z",
+     "start_time": "2021-09-06T12:46:08.282613Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'A': -90000, 'B': -10000, 'C': -16000}\n",
+      "on\n",
+      "on\n",
+      "on\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(controller.absolute_position())\n",
+    "print(controller.motor_a.on_off_status())\n",
+    "print(controller.motor_b.on_off_status())\n",
+    "print(controller.motor_c.on_off_status())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "c5aade6d",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:19:49.659764Z",
+     "start_time": "2021-09-06T12:19:49.649603Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "controller.motors_off()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e7e2eb5b",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-09-06T12:19:53.183446Z",
+     "start_time": "2021-09-06T12:19:53.163445Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "arm.set_arm_kinematics()  # sets default values of arm speed to be 100 micro meters per second,\n",
+    "                          # acceleration and deceleration to be 2048 micro meters per second square"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d73781eb",
+   "metadata": {},
+   "source": [
+    "Manually move the motors and take the needle head to the position where you want to set the origin. This will be the left bottom corner of the chip and run following commands."
```

Comment:
i think it's probably fine to talk about needles, but for a general audience it would help to know what kind of setting we are talking about here - what is a needed, what chip is in mind, etc etc.

Reply:
ok, will add this information